Packs/APIVoid/Integrations/APIVoid/README.md

```diff
@@ -0,0 +1,816 @@
+APIVoid wraps up a number of services such as ipvoid & urlvoid
+This integration was integrated and tested with version 01 of APIVoid
+
+## Configure APIVoid on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for APIVoid.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API KEY |  | True |
+    | Good Reputation (Percentage) | If the percentage of detections is BELOW this value, the indicator is considered GOOD | True |
```

Comment:
Is there an external reference from APIvoid about the `Percentage`? I suggest adding it to the `_description.md` file as well.

Reply:
^^ @AdrianaRoseDiaz a slight nudge :)

Reply:
The percentage is up for the user to decide, if the percentage of detections is bellow a value (for example, if less than 10% of users declared it as suspicious), the indicator is considered GOOD. So in essence, the score is not static to one thing, it depends on user scores.